### DISCUSSION TITLE
Stretch section for Flexbox Containers

### DISCUSSION THREAD

**Discussion by mymottow** [elementor/elementor#22783] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There's no option to "forced" stretch containers to full width like there was with the "old" sections

![Screenshot 2023-06-15 at 10 50 42](https://github.com/elementor/elementor/assets/83646243/86512b66-f1e0-496c-8fca-ba5ada8fd818)

### Describe the solution you'd like

Stretch section for Flexbox Containers

**Comment by toineenzo** [comment]
You can create the same effect when setting the Container to full width and setting the margin/padding to 0.
